src/Network/Request.php

```diff
@@ -893,7 +893,10 @@ public function here($base = true)
      */
     public function header($name)
     {
-        $name = 'HTTP_' . str_replace('-', '_', $name);
+        $name = str_replace('-', '_', $name);
+        if (strtoupper(substr($name, 0, 8)) !== 'CONTENT_') {
```

Comment:
Is the obsoleted `Content-MD5` header something that should be kept being supported? I've only seen it being set prefixed so far, ie as `HTTP_CONTENT_MD5`.